content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,106 @@
 Floors and tables management
 ============================
 
-Add a floor
-===========
+Floor plan
```

Comment:
I'd remove this title i think, so then you would have your intro paragraph right after the h1 title to explain the concept of a floor plan + your example (which is great btw!)